### ISSUE TITLE
Get userid on registration

### ISSUE THREAD

**Issue by solace** [meteor-useraccounts/core#612]
Hi,

Is there a hook or event that I can attach to after a user registers so that I can get the userId that was created?

`postSignUpHook` is only server side, but I need this information on the client. And `onSubmitHook` doesn't make this information available, and is also fired before.

Thanks

**Comment by cyclopse87** [comment]
@solace Was you able to find a solution for this?

**Comment by solace** [comment]
@cyclopse87 : Nope. But decided it wasn't a feature important enough to pursue, but in terms of polish, it would still be great to have.

**Comment by cyclopse87** [comment]
Thanks for reply
